johnwayodi/store-manager
# #161165586 View for Admin to Add New Product

#### What does this PR do?
Adds view that will allow the administrator to add details and create a new product
#### Description of Task to be completed?
Ensure product is created when all fields are filled and **_create product_** is clicked
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/store-manager.git`
2. Ensure you are in the directory where you cloned this repository, **_store-manager_** by default
3. `git checkout ft-add-product-admin-161165586`
4. open add-product.html on browser
#### Any background context you want to provide?
This page is available only to the Administrator user
#### What are the relevant pivotal tracker stories?
#161165586
#### Screenshot
![new-product](https://user-images.githubusercontent.com/10683903/46834903-60b43e80-cdb5-11e8-86d0-37b6ee92fb2d.png)

#### Questions:
Any feedback on what I can improve or change? Drop a comment or suggestion below
